juriscraper/lib/string_utils.py

```diff
@@ -182,7 +205,7 @@ def fix_camel_case(s):
         s_out = s
     else:
         s_out = s[0]
-        for i in xrange(1, len(s)):
+        for i in six.moves.range(1, len(s)):
```

Comment:
Similar to `iteritems`...yeah we shouldn't have an issue simplifying. I believe the catch is in Python 3.x `range` acts like `xrange` in that I think it's a lazy sequence while in Python 2.x `range` returns a list immediately. The length of `s` shouldn't be an issue here, right?